### ISSUE TITLE
Conditional resource still trigger planner events.

### ISSUE THREAD

**Issue by Typositoire** [sparkleformation/sfn#229]
SFN Version : sfn (3.0.20)
SparkeFormation Version : sparkle_formation (3.0.10)

```
  Update plan for: logstash-development-v2 > LogstashCacheCluster
    Resources to be replaced:
      [AWS::Route53::RecordSet]    RedisCname0Route53RecordSet    Reason: Updated properties: `HostedZoneName`
      [AWS::Route53::RecordSet]    RedisCname1Route53RecordSet    Reason: Updated properties: `HostedZoneName`
```

That was triggered by (with env_name being 'development')

```
conditions do
    is_production equals!(ref!(:env_name), 'production')
  end

dynamic!(:route53_record_set, "redis_cname_#{i}".to_sym) do
      properties do
        name "REDACTED"
        type "CNAME"
        TTL 60
        hosted_zone_name "REDACTED"
        resource_records [ attr!("elasticache_replication_group_#{i}".to_sym, "PrimaryEndPoint.Address") ]
      end
      on_condition! :is_production
    end
```

The update didn't do anything to route53 but the planner seems to believe it should have.

**Comment by Typositoire** [comment]
Ohh really ?? I was convinced it worked at some point? Thanks!

**Comment by slyness** [comment]
@chrisroberts are conditional plan related changes included in the latest release?

**Comment by chrisroberts** [comment]
Yes, see: #230

**Comment by slyness** [comment]
Awesome. Resolved by #230
